docs/2_4_common_schema.adoc

```diff
@@ -1919,6 +1919,11 @@ An importer may connect terminals if some or no terminal member variables are pr
 |Matching of the variables is based on the order of the terminal member variables.
 An importer should connect terminals only if the number of member variables matches.
 
+[[none,`none`]]
+|`none`
+|Matching of the variables is not intended by the creator of the FMU. 
+An importer may connect terminals based on the user's additional input.
```

Comment:
```suggestion
However, an importer may still connect terminals based on additional information.
```
The additional information might come from anywhere. I think we don't mention "the user" anywhere else and should avoid it here too.